sessionctx/stmtctx/stmtctx.go

```diff
@@ -27,11 +27,19 @@ import (
 type StatementContext struct {
 	// Set the following variables before execution
 
-	InInsertStmt           bool
-	InUpdateOrDeleteStmt   bool
-	InSelectStmt           bool
-	IgnoreTruncate         bool
-	IgnoreZeroInDate       bool
+	InInsertStmt         bool
+	InUpdateOrDeleteStmt bool
+	InSelectStmt         bool
+	IgnoreTruncate       bool
+	IgnoreZeroInDate     bool
+	// IgnoreErr is used for the following statements to avoid return an
+	// error, `insert`, `update`, and `delete`.
+	// If IgnoreErr is true and the statements meet some errors,
+	// e.g. duplicate keys, insert null to a not null column...,
+	// the errors will be appended as warnings.
+	// For the `load data` statement, IgnoreErr should be always true,
+	// because it is the same behavior as `insert ignore` statement.
+	IgnoreErr              bool
```

Comment:
maybe a better name is IgnoreWriteErr ?

Reply:
But simply naming it `IgnoreErr` is so confusing and may lead some misunderstanding

Reply:
So there is a comment for it.

Reply:
maybe `IgnoreDDLErr` is better? Per my understanding, `insert`, `update` and `delete` are ddl query.

Reply:
- DDL is short for "data definition language"
- DML is short for "data manipulation language"
I think DDLErr is still not a good name.

Reply:
I mean DML - - . Not DDL. DDL is just `create` table, `alter` table and etc.

Reply:
But `replace` statement could not ignore errors, and meanwhile `create table select` statement could ignore errors. The ignore behavior needs more check but not in this pr. I think it is wasting  time for the variable name.